t/maxconns.t

```diff
@@ -1,32 +1,42 @@
 #!/usr/bin/perl
-# NOTE: This test never worked. Memcached would ignore maxconns requests lower
-# than the current ulimit. Test needs to be updated.
 
 use strict;
 use warnings;
 
 use Test::More;
-
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use MemcachedTest;
 
-plan skip_all => "maxconns test does not work";
-exit 0;
-
-plan tests => 6;
-
-# start up a server with 10 maximum connections
-my $server = new_memcached('-c 100');
-my $sock = $server->sock;
-my @sockets;
-
-ok(defined($sock), 'Connection 0');
-push (@sockets, $sock);
-
+#my $server = new_memcached_nodebug('-c 30');
```

Comment:
can drop the nodebug changes since you're not using it anymore, yes?

Reply:
yes.